syntax = "proto2";

package boxbot.message;

message MotorVelocityMessage {
  required int32 right = 1;
  required int32 left = 2;
}